Cargo.toml

```diff
@@ -12,15 +12,22 @@ rustc-serialize = "0.3"
 docopt = "0.6"
 time = "0.1"
 ctrlc = { git = "https://github.com/tomusdrw/rust-ctrlc.git" }
+fdlimit = { path = "util/fdlimit" }
+daemonize = "0.2"
+number_prefix = "0.2"
 clippy = { version = "0.0.44", optional = true }
-ethcore-util = { path = "util" }
+
 ethcore = { path = "ethcore" }
+ethcore-util = { path = "util" }
 ethsync = { path = "sync" }
-ethcore-rpc = { path = "rpc", optional = true }
-fdlimit = { path = "util/fdlimit" }
-daemonize = "0.2"
 ethcore-devtools = { path = "devtools" }
```

Comment:
I think that it isn't. It should still be used when you run `cargo test --release`